### PR TITLE
#4976 Added missing pathogen test mappings

### LINKED ISSUE #4976: [DEMIS Interface] The DEMIS mapping does not apply when processing messages and only creating new pathogen test results
### Bug Description
When a user selects an existing sample and only adds a pathogen test result, the mapping of fields does not apply.
### Steps to Reproduce
1. Process a lab message
2. Choose an existing sample and only create a new pathogen test result

### Expected Behavior
The mapping should apply when only creating a new pathogen test result too
### Screenshots
**Sample + pathogen test result**

DemisMappingPathOnly2

**Pathogen test result**

DemisMappingPathOnly

### System Details
* Device:
* SORMAS version: 1.58.1
* DEMIS Adapter: 1.3.0
* Android version/Browser: firefox
* Server URL: test1402.sormas-oegd.de
* User Role: admin, national user

### PR DESCRIPTION
**Pull request by Alexandru-Pirvu**
Closes #4976

### CODE REVIEW

**Comment by FredrikSchaeferVitagroup** [comment]
This field needs to be checked, too.
image

File: sormas-ui/src/main/java/de/symeda/sormas/ui/labmessage/LabMessageController.java [hunk 1]
```diff
@@ -675,6 +675,10 @@ private void createPathogenTest(SampleDto sampleDto, LabMessageDto labMessageDto
 		Window window = VaadinUiUtil.createPopupWindow();
 		CommitDiscardWrapperComponent<PathogenTestForm> pathogenTestCreateComponent =
 			getPathogenTestCreateComponent(sampleDto, labMessageDto, pathogenTestDto, window);
+		CheckBox viaLimsCheckbox = pathogenTestCreateComponent.getWrappedComponent().getField(PathogenTestDto.VIA_LIMS);
+		viaLimsCheckbox.setValue(Boolean.TRUE);
```
Comment (FredrikSchaeferVitagroup) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/labmessage/LabMessageController.java:679]: [Here](https://github.com/hzi-braunschweig/SORMAS-Project/blob/9ea12de565dace01f2544b396be2acacf69c0535/sormas-ui/src/main/java/de/symeda/sormas/ui/labmessage/LabMessageController.java#L652) is a null check involved. Is there a reason why you ommited this check in your implementation?

Comment (Alexandru-Pirvu) [reply sormas-ui/src/main/java/de/symeda/sormas/ui/labmessage/LabMessageController.java:679]: The PathogenTestForm contains that field. I saw no reason to check for null.

Comment (FredrikSchaeferVitagroup) [reply sormas-ui/src/main/java/de/symeda/sormas/ui/labmessage/LabMessageController.java:679]: Does it always exist, also for non german systems?

Comment (FredrikSchaeferVitagroup) [reply sormas-ui/src/main/java/de/symeda/sormas/ui/labmessage/LabMessageController.java:679]: No, it does not.

Comment (FredrikSchaeferVitagroup) [reply sormas-ui/src/main/java/de/symeda/sormas/ui/labmessage/LabMessageController.java:679]: After a deeper look it seems that it always exists. I am wondering what the null check is there for...

**Review by FredrikSchaeferVitagroup — APPROVED** [review]

**Pull request merged**